libs/neo4j/pyproject.toml

```diff
@@ -27,6 +27,12 @@ pytest-socket = "^0.7.0"
 pytest-watcher = "^0.3.4"
 langchain-core = {git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/core"}
 
+[tool.poetry.group.langchain_openai]
+optional = true
+
+[tool.poetry.group.langchain_openai.dependencies]
+langchain-openai = "^0.2.8"
```

Comment:
We probably don't need this as an optional dependency if you've removed the example

Reply:
Removed this